docker/ray-ml/Dockerfile

```diff
@@ -23,7 +23,7 @@ RUN sudo apt-get update \
         unzip \
         unrar \
     && $HOME/anaconda3/bin/pip --no-cache-dir install -U pip \
-    && $HOME/anaconda3/bin/pip --no-cache-dir install -U \
+    && $HOME/anaconda3/bin/pip --no-cache-dir install --use-deprecated=legacy-resolver -U \
```

Comment:
can we not use the legacy resolver? what are the conflicts we are getting if we remove this?